django/conf/global_settings.py

```diff
@@ -667,3 +667,8 @@ def gettext_noop(s):
 SECURE_REFERRER_POLICY = "same-origin"
 SECURE_SSL_HOST = None
 SECURE_SSL_REDIRECT = False
+
+#########
+# TASKS #
+#########
+TASKS = {"default": {"BACKEND": "django.tasks.backends.immediate.ImmediateBackend"}}
```

Comment:
If it's explicitly set empty by the user, then any task related functionality would raise an error. I believe that's the same for databases?

Reply:
Hmm... maybe the comment on `DATABASES` is out of date. Please ignore my comment then!

Reply:
Looks like the comment is still accurate - `DATABASES` has a special case:

https://github.com/django/django/blob/857b1048d53ebf5fc5581c110e85c212b81ca83a/django/db/utils.py#L149-L150

I don't think that's necessary in this case, however I'll add a test so that at least setting it blank is semi-defined behaviour.